package.json

```diff
@@ -4,13 +4,14 @@
   "private": true,
   "main": "app.js",
   "scripts": {
-    "start": "babel-node ./src/app.js",
-    "dev": "babel-node ./src/app.js",
+    "start": "babel-node src/app.js",
```

Comment:
babel-node is not used in production. We'll use this now, but note that you have to change this before starting challenge 3.

Reply:
Ok noted